kythe/go/extractors/config/config_parser.go

```diff
@@ -145,9 +158,9 @@ func Load(r io.Reader) (*ecpb.ExtractionConfiguration, error) {
 
 // CreateImage uses the specified extraction configuration to generate a
 // new extraction image, which is written to the specified output path.
-func CreateImage(outputPath string, config *ecpb.ExtractionConfiguration) error {
+func CreateImage(outputPath string, config *ecpb.ExtractionConfiguration, repoDir, extractionOutDir string) error {
```

Comment:
The parameter ordering is kind of weird here. Can we get some kind of convention about the order of these things? Maybe "config first" or something?